packages/dashboard/src/customization/widgets/lineChart/component.tsx

```diff
@@ -41,6 +43,18 @@ const LineChartWidgetComponent: React.FC<LineChartWidget> = (widget) => {
   // the 44 is from the widget tile header's height
   const size = { width: chartSize.width, height: chartSize.height - 44 };
 
+  /* condition to check if the widget is empty or not based on assets length while adding and removing assets */
+  const isEmptyWidget = queryConfig.query ? queryConfig.query.assets.length === 0 : true;
```

Comment:
`const isEmptyWidget = !queryConfig.query ?? queryConfig.query.assets.length === 0;`

Reply:
thanks for the feedback, small correction here 
const isEmptyWidget = !queryConfig.query || queryConfig.query.assets.length === 0;